src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -464,7 +464,7 @@ def load_command_table(self, _):
         g.wait_command('wait')
 
     with self.command_group('ppg', compute_proximity_placement_groups_sdk, min_api='2018-04-01', client_factory=cf_proximity_placement_groups) as g:
-        g.command('show', 'get')
+        g.custom_command('show', 'show_proximity_placement_group')
```

Comment:
just one question. Should we use g.custom_command? Based on your implementation, I think we can still use command.

Reply:
well. I got your point. It's a required filed. Right? I think you can also set `required=False` to use it. But this one is fine.

Reply:
looks like if this filed is not required in SDK. I don't think you need change this one. You can just rewrite the c.argument.

Reply:
I agree. We don't need to use `custom_command`. Also we should use `show_command` and `custom_show_command` so that the [exit code](https://github.com/Azure/azure-cli#exit-codes) is correctly set to 3 on a 404.

Reply:
@MyronFanQiu Agree. command() is enough. I didn't realize that there is such an argument. Then I added a new one.

Reply:
@jiasli Good advice. An interesting fact is that there is no show_command() is vm module's params. I'll try it.